.github/workflows/pr.yaml

```diff
@@ -8,4 +8,6 @@ on:
 jobs:
   shared-operator-workflow:
     name: shared-operator-workflow
-    uses: redhat-cop/github-workflows-operators/.github/workflows/pr-operator.yml@v1.0.0
+    uses: trevorbox/github-workflows-operators/.github/workflows/pr-operator.yml@main
```

Comment:
@raffaelespazzoli See https://github.com/redhat-cop/github-workflows-operators/pull/7 
I think there is an opportunity to update the shared workflow to add testing.